interface.html

```diff
@@ -20,7 +20,7 @@
               <button data-show-alive-source class="active-source btn-round btn">All</button>
               <button data-show-trash-source class="btn-round spaced btn">Trash</button>
               <input class="search input-round pull-right form-control" type="search" placeholder="Search data sources" />
-              <a href="https://help.fliplet.com/data-sources/" class="help-icon" target="_blank">
+              <a data-show-help-link href="https://help.fliplet.com/data-sources/" class="help-icon" target="_blank">
```

Comment:
Please remove the word `show` from the data attribute

Reply:
@squallstar Fixed